app/src/newdocs/content/streams.mdx

```diff
@@ -19,7 +33,61 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 <ScrollableAnchor id="work-with-streams-via-sdks"><div>
 
 ## Work with streams via SDKs
---content--
+
+The easiest way to work with streams is to use the JavaScript client, which works in `node.js` as well as the browser. See the <Link to={links.newdocs.SDKs}>SDKs section</Link> for the official and community-maintained client libraries available for various languages:
+
+- **JavaScript** client works in the browser as well as node.js.
+- **Java** client is work-in-progress but already implements the most important set of functionality.
+
+Client libraries for other languages are on the <a href={links.community.trello} target="_blank" rel="nofollow noopener noreferrer">roadmap</a>. If there isn't a client library available for your language, you can dive into the details of the [websocket protocol](https://github.com/streamr-dev/streamr-client-protocol-js/blob/master/PROTOCOL.md) 
```

Comment:
> If there isn't a client library available for your language

maybe:

> If a client library isn't available for your language

Reply:
also I have concerns about linking to `master` vs linking to a commit, if `PROTOCOL.md` ever moves then this link breaks, while a link like: https://github.com/streamr-dev/streamr-client-protocol-js/blob/bb2f0168bcff454b5c458e275e9c2de0dee342f9/PROTOCOL.md will always point at the same place, though downside there is that it won't automatically update as `PROTOCOL.md` is updated. It's a tradeoff but I  hit broken links to things in blob/master all the time and it's very annoying.

Reply:
Yep, permanent link is perhaps better.